spec/v3/services/log/find_spec.rb

```diff
@@ -154,13 +175,61 @@
 
     describe 'it returns the correct content type' do
       example do
-        s3log.update_attributes(archived_at: Time.now)
+        s3log.update_attributes(archived_at: Time.now, archive_verified: true)
```

Comment:
The code that checks for the field is also changed to be consistent with how `#archiving?` is used in V2.